code/game/objects/structures/noticeboard.dm

```diff
@@ -25,7 +25,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/noticeboard, 32)
 		if(istype(I, /obj/item/paper))
 			I.forceMove(src)
 			notices++
-	icon_state = "nboard0[notices]"
+	update_icon()
```

Comment:
shouldn't we use `update_appearance()` here (which will invoke `update_icon()`)? this is just in case someone wants to be cute and have the description update with how many notices are on the board or something like that in the future.

Reply:
yes, or well -
```suggestion
	update_appearance(UPDATE_ICON)
```